rpcs3/CMakeLists.txt

```diff
@@ -166,6 +166,26 @@ find_package(OpenGL REQUIRED)
 find_package(OpenAL REQUIRED)
 if (NOT WITHOUT_LLVM)
 	find_package(LLVM 4.0 CONFIG)
+  if (NOT LLVM_FOUND)
+    message("System LLVM was not found, LLVM will be built from the submodule.")
+
+    set(LLVM_TARGETS_TO_BUILD "X86")
```

Comment:
Change this line to
```cmake
set(LLVM_TARGETS_TO_BUILD "X86" CACHE INTERNAL "")
```
which from my testing should work, and will shorten built time immensely.

Reply:
Done. Thanks for figuring out. This got rid of all the libraries we don't need.